doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1545 @@
+    sopmd_avg                    = FLOAT                            ; second order polarization mode dispersion avg
+    sopmd_min                    = FLOAT                            ; second order polarization mode dispersion min
+    sopmd_max                    = FLOAT                            ; second order polarization mode dispersion max
+    pdl_avg                      = FLOAT                            ; polarization dependent loss avg
+    pdl_min                      = FLOAT                            ; polarization dependent loss min
+    pdl_max                      = FLOAT                            ; polarization dependent loss max
+    osnr_avg                     = FLOAT                            ; optical signal to noise ratio avg
+    osnr_min                     = FLOAT                            ; optical signal to noise ratio min
+    osnr_max                     = FLOAT                            ; optical signal to noise ratio max
+    esnr_avg                     = FLOAT                            ; electrical signal to noise ratio avg
+    esnr_min                     = FLOAT                            ; electrical signal to noise ratio min
+    esnr_max                     = FLOAT                            ; electrical signal to noise ratio max
+    cfo_avg                      = FLOAT                            ; carrier frequency offset avg
+    cfo_min                      = FLOAT                            ; carrier frequency offset min
+    cfo_max                      = FLOAT                            ; carrier frequency offset max
+    soproc_avg                   = FLOAT                            ; state of polarization rate of change avg
+    soproc_min                   = FLOAT                            ; state of polarization rate of change min
+    soproc_max                   = FLOAT                            ; state of polarization rate of change max
+    tx_power_avg                 = FLOAT                            ; tx output power avg
+    tx_power_min                 = FLOAT                            ; tx output power min
+    tx_power_max                 = FLOAT                            ; tx output power max
+    rx_tot_power_avg             = FLOAT                            ; rx total power avg
+    rx_tot_power_min             = FLOAT                            ; rx total power min
+    rx_tot_power_max             = FLOAT                            ; rx total power max
+    rx_sig_power_avg             = FLOAT                            ; rx signal power avg
+    rx_sig_power_min             = FLOAT                            ; rx signal power min
+    rx_sig_power_max             = FLOAT                            ; rx signal power max 
+
+##### 2.1.6 Transceiver Loopback Table #####
+
+    ; Defines Transceiver loopback information for a port
+    key                          = TRANSCEIVER_PM|ifname            ; information of loopback on port
```

Comment:
Shouldn't 'key' be 'TRANSCEIVER_LOPBACK|ifname' here instead of 'TRANSCEIVER_PM|ifname'?

Also, 'TRANSCEIVER_STATUS|ifname' is already a key in section 2.1.4 (which is correct)

Reply:
modified.